src/components/custom-aggrid/custom-aggrid-filters/custom-aggrid-duration-filter.tsx

```diff
@@ -0,0 +1,142 @@
+/**
+ * Copyright (c) 2024, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+import React, { useEffect, useState } from 'react';
+import {
+    Grid,
+    InputAdornment,
+    TextField,
+    Typography,
+    IconButton,
+} from '@mui/material';
+import { useIntl } from 'react-intl';
+import ClearIcon from '@mui/icons-material/Clear';
+
+const styles = {
+    iconSize: {
+        fontSize: '1rem',
+        padding: '0',
+    },
+    flexCenter: {
+        display: 'flex',
+        alignItems: 'center',
+        justifyContent: 'center',
+    },
+    noArrows: {
+        '& input::-webkit-outer-spin-button, & input::-webkit-inner-spin-button':
+            {
+                display: 'none',
+            },
+        '& input[type=number]': {
+            MozAppearance: 'textfield',
+        },
+    },
+};
+
+interface ICustomAggridDurationFilter {
+    value: string | undefined; // duration in seconds as a string
+    onChange: (value: string | undefined) => void;
+}
+
+const CustomAggridDurationFilter: React.FC<ICustomAggridDurationFilter> = ({
+    value,
+    onChange,
+}) => {
+    const intl = useIntl();
+    const [minutes, setMinutes] = useState('');
+    const [seconds, setSeconds] = useState('');
```

Comment:
We send a string value to the backend; otherwise, we will encounter an error. I chose to use a string to distinguish between an empty string and '0' for resetting purposes.

Reply:
I would have use `null` or `undefined` to distinguish that.